Lab-2.ipynb

```diff
@@ -0,0 +1,623 @@
+      "text/plain": [
+       "<Figure size 2016x504 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "p5 = convolve(\"Poisson\",5,Xbkg)\n",
+    "plotter(\"Poisson\",p5,5,\"sum\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These graphs plot the same Poisson distribution with a mean of X = 2 summed 5 times now, and the corresponding semilog plot. This means we have summed the average cosmic ray counts for 5 separate days, using the average from cosmic ray counts per day. The left graph shows that after 5 summations, the distribution still retains the discrete Poisson shape, and that the probability to measure the average cosmic ray count for 5 days decreased. The semilog plot demonstrates that the distribution has not approached a Gaussian shape, similar to the previous problem. We expect a parabolic shape if it had approached a Gaussian distribution.\n",
+    "\n",
+    "Conceptually, this makes sense. Here we are only looking at expected cosmic ray counts each day, so we shouldn't expected the distribution of the background noise to approach a continuous distribution unless we calculate how the mean changes in a 1-day period by an averaging convolution operation.\n",
+    "\n",
+    "Mathematically, this outcome is expected as well. Summing the average counts from one day over a 5-day observation period, is identical to summing 5 days' average counts for a single period. This operation just adds the expected values over time. We aren't dividing (such as in an averaging operation) so there is no scaling process happening, which would alter the shape in that process."
```

Comment:
1.B: I enjoyed your wording for the mathematical part. Better than how I worded it. Nice.